src/service/loading-indicator.js

```diff
@@ -68,12 +67,9 @@ export class LoadingIndicatorImpl {
       : null);
     let io;
     try {
-      io = new win.IntersectionObserver(ioCallback, {
-        root,
-        rootMargin,
-      });
+      io = new win.IntersectionObserver(ioCallback, {root, rootMargin: '25%'});
     } catch (e) {
-      io = new win.IntersectionObserver(ioCallback, {rootMargin});
+      io = new win.IntersectionObserver(ioCallback, {rootMargin: '150px'});
```

Comment:
Nit: Can we use `25vw 25vh`?

Reply:
why would this be allowed but `25%` be banned?

Reply:
Trying to confirm what we can and cannot use as values for those browsers. So far no luck reproducing it even on Chromium  50.0.2661.0. It's possible that this is an old Android WebView that's WebKit based.

Reply:
Keeping `px` for max performance. `150px` is a good default around 25%.